server/controllers/calendarController.ts

```diff
@@ -19,10 +19,12 @@ export default class CalendarController {
 
   async getDate(req: Request, res: Response) {
     const {
-      user: { token, username },
+      user,
       params: { date },
     } = req
 
+    if (!user) return false
```

Comment:
annoyingly `user` can now be not defined, even though our middleware enforces it anyway.  Therefore need this strange `if` check.  Tried a `!!` but that is rejected by the linting too...